packages/app-page-builder/src/editor/plugins/elements/pagesList/graphql.ts

```diff
@@ -39,13 +32,7 @@ export const LIST_PUBLISHED_PAGES = gql`
                     }
                 }
                 meta {
-                    from
-                    to
-                    page
-                    nextPage
-                    previousPage
                     totalCount
```

Comment:
Also add `cursor` and `hasMoreItems` properties in `meta`. They will be useful for implementing pagination :)